src/tree/tests/test_resolution.rs

```diff
@@ -89,7 +89,7 @@ fn test_original_child_resolution() {
         // Resolution for root left child
         const LEFT_CHILD_RESOLUTION: &[usize] = &[0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14];
         // Arbitrary unmerged leaves for root
-        const ROOT_UNMERGED_LEAVES: &[u32] = &[4, 6, 8];
+        const ROOT_UNMERGED_LEAVES: &[u32] = &[2, 3, 4];
```

Comment:
3 should be an invalid `NodeIndex` if you want to index a leaf

Reply:
But `unmerged_leaves` are now  `Vec<LeafIndex>`, so 2, 3 and 4 are now the indices corresponding to the earlier 4, 6 and 8.

Reply:
Right, that makes sense now.